app/controllers/apipie/apipies_controller.rb

```diff
@@ -14,11 +14,43 @@ def authenticate
       end
     end
 
+
+    def render_swagger_json
+      # the 'authorize' configuration is not supported for swagger output
+      head :forbidden and return if Apipie.configuration.authorize
+
+      @language = get_language
+
+      Apipie.load_documentation if Apipie.configuration.reload_controllers? || (Rails.version.to_i >= 4.0 && !Rails.application.config.eager_load)
```

Comment:
There is quite a lot of code duplicated from the `index` method. Could you look into how to DRY this a bit?

Otherwise, the implementation looks good and I think we are close to merging this